resources/setup.sh

```diff
@@ -68,12 +68,13 @@ fi
 
 # run kubeadm to create cluster - ignore preflights as there will be failures because of swap, systemd, lots of things..
 if [ ! -f /var/lib/kubeadm.yaml ]; then
-    cp /var/tmp/minikube/kubeadm.yaml /var/lib/kubeadm.yaml
+    cp /var/tmp/minikube/kubeadm.yaml.new /var/lib/kubeadm.yaml
```

Comment:
could you make this backward compat by adding an `if [ -f /var/tmp/minikube/kubeadm.yaml ]` else `.new` etc